components/content-topic-one.js

```diff
@@ -0,0 +1,42 @@
+import styled from 'styled-components';
+
+export default function Headline() {
+  return (
+    <>
+      <HeadLine>Contentheadline Topic 1</HeadLine>
+    </>
+  );
+}
+
+export function ContentTopicStepOne() {
+  return (
+    <>
+      <Steps>
+        Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo
+        ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis
+        dis parturient montes, nascetur ridiculus mus. Donec quam felis.
+      </Steps>
+    </>
+  );
+}
+
+export function ContentTopicStepTwo() {
+  return (
+    <>
+      <Steps>
+        <div>
+          Bassa quis enim. Donec pede justo, fringilla vel, aliquet nec,
+          vulputate eget, arcu. In enim justo, rhoncus ut, imperdiet a,
+          venenatis vitae, justo. Nullam dictum felis eu pede mollis pretium.
+          Integer tincidunt.
+        </div>
+      </Steps>
+    </>
+  );
+}
```

Comment:
Each component-file should only contain one component, not several.
In this case, you should create three components.

Or since each component is rather small, summarise them in one component.

Reply:
I did not find any other solution yet to add the pictures between the steps. And if I add every step to a single component, the app gets really fast very full and cluttery.

Reply:
That is a good point.

However, having reusable components actually solves the issue of your App getting too full/cluttery when you only need to change small parts of the respective component.

Having a component that for instance only has a `<div>` most likely does not need to be a component itself, but can be part of a bigger component instead.

But since you say that this is an intermediate solution, you might leave it as it is for the moment.
We can also meet up and think about a solution together.